vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -88,6 +88,11 @@ stress_test_arguments
 ### Understanding the input arguments
 The meaning of each of the detail arguments is as follows:
 
+* `baseline_scenario` holds the name of the baseline scenario to be used in the
+stress test. We use a business-as-usual scenario for the baseline.
+* `shock_scenario`holds the name of the scenario that companies are forced to
```

Comment:
missed a space after the variable name. Not trying to be overly picky, up to you to fix before merging